core/btdiscovery.cpp

```diff
@@ -31,8 +31,8 @@ static dc_descriptor_t *getDeviceType(QString btName)
 		else if (btName.mid(4,2) == "2-") product = "OSTC 2N";
 		else if (btName.mid(4,2) == "+ ") product = "OSTC 2";
 		// all OSTCs are HW_FAMILY_OSTC_3, so when we do not know,
-		// just try this
-		else product = "OSTC 3"; // all OSTCs are HW_FAMILY_OSTC_3
```

Comment:
Yea, the "OSTC 3" didn't have bluetooth, so thats probably why it fell away somewhere along the way. The "OSTC Plus" (3+) all as bluetooth.

Reply:
I kept it as "OSTC 2" as the default product name = I saw no advantage naming it "OSTC Plus"
But I updated the comment to be clearer.